src/bitswap/wantlist.js

```diff
@@ -5,42 +5,58 @@
 const { getDescribe, getIt, expect } = require('../utils/mocha')
 const { waitForWantlistKey } = require('./utils')
 
+const TIMEOUT = 120000
+
 module.exports = (createCommon, options) => {
   const describe = getDescribe(options)
   const it = getIt(options)
   const common = createCommon()
 
-  describe('.bitswap.wantlist', function () {
-    this.timeout(60 * 1000)
+  describe('.bitswap.wantlist', () => {
     let ipfsA
     let ipfsB
     const key = 'QmUBdnXXPyoDFXj3Hj39dNJ5VkN3QFRskXxcGaYFBB8CNR'
 
-    before(async () => {
+    before(async function () {
+      // CI takes longer to instantiate the daemon, so we need to increase the
+      // timeout for the before step
+      this.timeout(60 * 1000)
+
       ipfsA = await common.setup()
-      ipfsB = await common.setup()
+      ipfsB = await common.setup({ factoryOptions: { type: 'go' } })
```

Comment:
Why is this specifying go?

Reply:
This was a change trying to fix a failing test (only on CI, the test is passing locally). But I'll remove it because it didn't solve the problem.